features/rules/GRF/GRF007_valid-vertical-datum-crs-type.feature

```diff
@@ -0,0 +1,27 @@
+@implementer_agreement
+@GRF
+@version1
+@E00500
+Feature: GRF007 - Valid vertical datum CRS type
+The rule verifies that any coordinate reference system (CRS) assigned as a vertical datum is either a vertical CRS or a compound CRS containing a vertical component, to ensure valid height referencing.
+https://pyproj4.github.io/pyproj/stable/api/crs/crs.html
+
+
+  Scenario Outline: WKT specification for missing ESPG in the name
+
+      Given A model with Schema 'IFC4.3'
+      Given an .IfcProject.
+      Given Its attribute .RepresentationContexts.
+      Given Its attribute .HasCoordinateOperation.
+      Given Its attribute .TargetCRS.
+      Given Its attribute .VerticalDatum.
+      Given The value refers to a valid EPSG code
+      Given The CRS corresponding to the given EPSG code
+      Given .<vertical_or_compound>. ^is^ empty
+
+      Then .<compound_or_vertical>. ^is not^ empty
```

Comment:
I find this a bit hard to read. Can't we implement it in a natural language text without 'leaking' pyproj details?

```suggestion
      Then The CRS should define a vertical component
```

(or sth like that).

Maybe also skip the step "The CRS corresponding to the given EPSG code" because it's another data type to deal with in terms of having it in our stack and serializing it to the frontend.

Reply:
[Done](https://github.com/buildingSMART/ifc-gherkin-rules/pull/413/commits/a847cd1019bafb5158a1f62b43689fc97f2af199)